app/templates/project-name.Gulp/tasks/watch.js

```diff
@@ -1,38 +1,36 @@
+import gulp from 'gulp';
 import browserSync from 'browser-sync';
 import cfonts from 'cfonts';
-import gulp from 'gulp';
 import config from '../config';
 
 function watch() {
     cfonts.say(config.pkg.name, {
-        font: 'simple',
-        letterSpacing: 0,
-        space: false,
-        maxLength: '20'
+        'font': 'simple',
+        'letterSpacing': 0,
+        'space': false,
+        'maxLength': '20'
     });
 
-    gulp.watch(`${config.css.src}/**/*.scss`).on('change', () => {
+    gulp.watch(`${config.css.src}/**/*.scss`).on('change', function() {
```

Comment:
Hi, is there any reason for using scoped `function()` instead of `() => {}`? Just so I know 😄 Thanks

Reply:
Nope just me being silly and overlooking one 😁